pulser/devices/_device_datacls.py

```diff
@@ -111,7 +111,9 @@ def rydberg_blockade_radius(self, rabi_frequency: float) -> float:
         Returns:
             float: The rydberg blockade radius, in μm.
         """
-        return (self.interaction_coeff / rabi_frequency) ** (1 / 6)
+        return cast(
```

Comment:
I tried this on my computer and mypy is not returning an error if I omit this cast. However, it's raising the error that there's a redundant cast to float now, on the most commit of develop (e0f7849a1747523e1503f008e5c2a050cbb22086). @HGSilveri

Reply:
It seems that there has been a yet another release (0.931) of mypy on the 7th of January. This change was made with 0.930, so that might be why. However, I just updated to 0.931 and didn't get this error you mention. What `mypy` version do you have locally?

In any case, I think it's clear that we have to pick a `mypy` version and stick with it.

Reply:
I'm using mypy 0.920

Reply:
Ok, I confirm that I can replicate the behaviour with that version. The same goes for 0.921.

I didn't like that mypy had me do this cast, so perhaps I'll fix the mypy version to 0.921 (the next one is already 0.930) and revert this change. What do you thinK?

Reply:
I think it's a bug, because storing the result in an intermediate float variable before returning it works with the 0.931 version. I agree that it would be easier to stick on 0.921 for now and consider later to update if we need new features and the bug has been fixed.